go/src/koding/kites/config/konfig.go

```diff
@@ -32,24 +23,66 @@ var KonfigCache = &CacheOptions{
 type Konfig struct {
 	// Kite configuration.
 	KiteKeyFile string `json:"kiteKeyFile,omitempty"`
+	KiteKey     string `json:"kiteKey,omitempty"`
 
 	// Koding endpoints konfiguration.
 	KontrolURL string `json:"kontrolURL,omitempty"`
 	KlientURL  string `json:"klientURL,omitempty"`
 	KloudURL   string `json:"kloudURL,omitempty"`
 	TunnelURL  string `json:"tunnelURL,omitempty"`
+	IPURL      string `json:"ipURL,omitempty"`
+	IPCheckURL string `json:"ipCheckURL,omitempty"`
 
 	// Klient / KD auto-update endpoints.
 	KlientLatestURL string `json:"klientLatestURL,omitempty"`
 	KDLatestURL     string `json:"kdLatestURL,omitempty"`
 
-	Debug bool `json:"debug,omitempty"`
+	// Public S3 bucket for writing logs.
+	PublicBucketName   string `json:"publicBucketName,omitempty"`
+	PublicBucketRegion string `json:"publicBucketRegion,omitempty"`
+
+	Debug int `json:"debug,omitempty"`
```

Comment:
also i realized that there are other occurrences of using integer for debug statement, could you fix them too?